README.md

```diff
@@ -24,10 +24,50 @@ All contracts on Chainlink must be funded using the local utility token LINK.
 
 ## Network Details
 
-Chainlink oracle address: 0x17dED59fCd940F0a40462D52AAcD11493C6D8073
-Job ID:
+### Ethereum Mainnet (Chain ID: 1)
 
-- indexName: cd7bea47b73a4d47a5cedecf00b9b561
-- indexValue: d06b539cdbeb450aa72f2e70223078af
-- aDayChange: 3d2a6108056644c38f98d755536eb5ac
-- aMonthChange: b871377caf1f4ccb851189f8d906c277
+- Payment amount: 0.01 LINK
+- Oracle address: 0xfE2dD37BC29f5fc4E0cad8F58F4Dbf4AddD5A59A
+- LINK token address: 0x514910771af9ca656af840dff83e8264ecf986ca
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
```

Comment:
Do we only have one job ID now?

Reply:
@JGJP yes, the job will return those 4 parameters together. You can check similar example here
https://github.com/truflation/quickstart/blob/nft-index/NftIndexTester.sol#L48-L57